app/views/templates/inputs/radioGroup.scala.html

```diff
@@ -0,0 +1,52 @@
+@*
+ * Copyright 2019 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import views.html.helper._
+
+@(field: Field,
+        choices: Seq[(String, String)],
+        question: String,
+        additionalContent: Option[Html] = None)(implicit messages: Messages)
+
+@elements = @{ FieldElements(field.id, field, null, Map(), messages) }
+
+<div @if(elements.hasErrors){ class="form-field--error"}>
+
+  <fieldset>
+    <legend class="visuallyhidden">
```

Comment:
Do we want this to be visually hidden? To avoid having the accessibility issue of multiple h1's it might be easier to make this visible

Reply:
Yep, this was raised in the DAC report